modules/nested-accordion/assets/js/frontend/handlers/nested-accordion.js

```diff
@@ -34,4 +50,123 @@ export default class NestedAccordion extends Base {
 			$items[ index ].appendChild( element );
 		} );
 	}
+
+	applyDefaultStateCondition() {
+		if ( ! this.elements ) {
+			return;
+		}
+
+		const { $titles, $items, $accordionContent } = this.getDefaultElements(),
+			{ default_state: currentState } = this.getElementSettings(),
+			{ default_state: defaultState } = this.getDefaultSettings();
+
+		if ( currentState === defaultState ) {
+			this.open( $items[ 0 ], $titles[ 0 ], $accordionContent[ 0 ] );
+		} else {
+			$items.each( ( _, item ) => item.removeAttribute( 'open' ) );
+		}
+	}
+
+	bindEvents() {
+		this.bindAnimationListeners();
+	}
+
+	unbindEvents() {
+		this.removeAnimationListeners();
+	}
+
+	bindAnimationListeners() {
+		const { $titles, $items, $accordionContent } = this.getDefaultElements();
+
+		$titles.each( ( index, title ) => {
+			title.addEventListener( 'click', ( e ) => {
+				this.clickListener( e, $items, $accordionContent, index, title );
+			} );
+		} );
+	}
+
+	clickListener( e, $items, $accordionContent, index, title ) {
+		e.preventDefault();
+
+		const detailsNode = $items[ index ],
```

Comment:
I suggest being consistent regarding var names,
`detailsNode` becomes `el` later on.
In both cases, we are talking about the item so maybe we can call it `item` or `itemContainer`.
Especially when we are talking about a single element that comes from an array called `items` :)